prompb/types.proto

```diff
@@ -56,6 +56,9 @@ message ReadHints {
   string func = 2;    // String representation of surrounding function or aggregation.
   int64 start_ms = 3; // Start time in milliseconds.
   int64 end_ms = 4;   // End time in milliseconds.
+  repeated string grouping = 5; // List of label names of any fields used in aggregation.
```

Comment:
s/of any fields/

PromQL doesn't have fields.

Reply:
What about `List of label names used in aggregation. `?